publish_release_rpms

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+
+. settings
+
+USER='rpmrepo'
+HOST='rpm.theforeman.org'
```

Comment:
You need to list the host explicitly. `rpm.theforeman.org` will be the CDN and you can't SSH into that.